server/api/entryRouters.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Request, Response, NextFunction } from "express";
+import { Entry } from "../db/index.js";
+import { EntryAttributes } from "../db/models/Entry.model.js";
+const router = express.Router();
+
+router.get("/", async (req: Request, res: Response, next: NextFunction) : Promise<void>=> {
+  try {
+    const entries: EntryAttributes[] = await Entry.findAll();
+    res.send(entries);
+  } catch (err) {
+    res.sendStatus(404);
+    next(err);
+  }
+});
+
+router.post("/", async (req: Request, res: Response, next: NextFunction): Promise<void>=> {
+    try {
+      const {entryType, date, creditDebit, amount, title, note, frequency} : EntryAttributes = req.body
+      // const createEntry= 
+      await Entry.create({
+        entryType, date, creditDebit, amount, title, note, frequency
+      })
+      res.sendStatus(204)
+    } catch (err) {
+      res.sendStatus(404);
+      next(err);
+    }
+  });
+  
+  router.delete("/:entryId", async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const entryId: string = req.params.entryId
```

Comment:
The id will be the uuid created on the backend when it is created. May prevent this error.